src/Shared/TaskToApm.cs

```diff
@@ -1,6 +1,5 @@
-// Licensed to the .NET Foundation under one or more agreements.
-// The .NET Foundation licenses this file to you under the MIT license.
-// See the LICENSE file in the project root for more information.
```

Comment:
So these were taken from CoreFX I think, hence we kept their license. Assuming that we'll be switching them all back to the MIT header shortly I think this is OK, but I want to raise them as a possible oddity.

Reply:
Yes, the next commit changes everything here plus ~7200 other files to use the MIT header. Normalizing the headers before the change makes that one _much_ easier to review because all changed files are changed in exactly the same way.